src/main/java/homeworks/daniel/secondAssigment/ClassA.java

```diff
@@ -0,0 +1,12 @@
+package homeworks.daniel.secondAssigment;
+
+import java.util.List;
+
+public class ClassA extends ClassABC {
+    private List c;
+
+    @Override
+    public void methodB(){System.out.println("ClassA:methodB");}
```

Comment:
Only those methods with unique (different from the parent) behaviors should be overridden.